settings.yml

```
@@ -80,7 +80,7 @@ variables:
   ##   Connection settings   ##
   #############################
 
-  serviceConnection: 'CARML-CSU-Tenant-Connection'
+  serviceConnection: 'skhan-service-connection'
```

Please revert back as a last after retesting
```suggestion
  serviceConnection: 'CARML-CSU-Tenant-Connection'
```